LanguageFeatures/primary-constructors/grammar_A02_t02.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Let `D` be a class, extension type, or enum declaration.
+///
+/// A compile-time error occurs if `D` includes a `<classNamePart>` that
+/// contains a `<primaryConstructorNoConst>`, and the body of `D` contains a
+/// `<declaringConstructorSignature>` that contains a `<declaringParameterList>`.
+///
+/// @description Check that it is a compile-time error if `D` includes a
+/// `<classNamePart>` that contains a `<primaryConstructorNoConst>`, and the
+/// body of `D` contains a `<declaringConstructorSignature>` that contains a
+/// `<declaringParameterList>`. Test an extension type.
+/// @author sgrekhov22@gmail.com
+
+// TODO (sgrekhov) Add `declaring-constructors` experimental flag
+
+extension type ET1(var int v) {
```

Comment:
`var` should be an error: A plain `int v` means `final int v` in the special case where the declaration is an extension type, and an extension type must declare exactly one final instance variable, so if we try to declare a non-final one then it's an error.